de.json

```
@@ -0,0 +1,847 @@
+        "Importer.Model.Advanced.AlbedoColor" : "Albedo Farb-Texturen importieren",
+        "Importer.Model.Advanced.ImportEmissive" : "Emissions Texturen importieren",
+
+        "Importer.Model.Advanced.Colliders" : "Collider generieren",
+        "Importer.Model.Advanced.Animations" : "Animationen importieren",
+        "Importer.Model.Advanced.SetupSnappable" : "Als einrastbare Objekte importieren",
+        "Importer.Model.Advanced.SetupTimelapse" : "Als Zeitraffer importieren",
+        "Importer.Model.Advanced.ImportExternalTextures" : "Externe Texturen importieren",
+        "Importer.Model.Advanced.SkinnedMeshes" : "Skinned Meshes importieren",
+        "Importer.Model.Advanced.SetupIK" : "Inverse Kinematik generieren",
+        "Importer.Model.Advanced.VisualizeRig" : "Rig visualisieren",
+        "Importer.Model.Advanced.ForceTpose" : "T-Pose forcieren",
+        "Importer.Model.Advanced.ForcePointCloud" : "Als Punkt Wolke",
+
+        "Importer.Model.Advanced.DualSided" : "Doppelseitiges Objekt",
+        "Importer.Model.Advanced.FlatShaded" : "Flach schattiert",
+        "Importer.Model.Advanced.DeduplicateInstances" : "Deduplizierung von Instanzen (langsam)",
+        "Importer.Model.Advanced.Optimize" : "Modell/Szene optimieren",
+        "Importer.Model.Advanced.RandomColors" : "Zufällige Farben",
+        "Importer.Model.Advanced.SpawnMaterialOrbs" : "Spawnen von Material-Orbs",
+        "Importer.Model.Advanced.ImagesByName" : "Bilder nach Namen importieren",
+
+        "Importer.Model.Advanced.Grabbable" : "Greifbar setzen",
+        "Importer.Model.Advanced.Scalable" : "Skalierbar setzen",
+        "Importer.Model.Advanced.PositionAtOrigin" : "Positionierung am Urspungspunkt",
+        "Importer.Model.Advanced.AssetsOnObject" : "Plaziere Assets auf Objekt",
+
+        "Importer.Video.Title" : "Video Importer",
+        "Importer.Video.Prompt" : "Was importieren Sie?",
+        "Importer.Video.Regular" : "Video",
+        "Importer.Video.360" : "360 Video",
+        "Importer.Video.Stereo" : "Stereo Video",
+        "Importer.Video.Stereo360" : "Stereo 360 Video",
+        "Importer.Video.Depth" : "Tiefen Video",
+        "Importer.Video.180" : "180 Video",
+        "Importer.Video.Stereo180" : "Stereo 180 Video",
+        "Importer.Video.Depth.Prompt" : "Tiefen Voreinstellung",
+        "Importer.Video.Depth.Default" : "Default",
+
+        "Credits.Title" : "Sende Betrag",
+        "Credits.TokenPrompt" : "Welches Token möchtest Du senden?",
+        "Credits.NCR.Title" : "Neos Credits (<b>NCR</b>)",
+        "Credits.NCR.Description" : "Ethereum Token mit realem Wert",
+        "Credits.KFC.Title" : "Kompletely Fake Credits (<b>KFC</b>)",
+        "Credits.KFC.Description" : "Neos' Sandbox Token ohne realen Wert.",
+        "Credits.AmountPrompt" : "{action,select, send {How much do you want to send?} withdraw {How much do you want to withdraw?} tip {How much do you want to tip?}}",

```

This one is missing translation, is this intentional?

---

```suggestion
        "Credits.AmountPrompt" : "{action,select, send {Wie viel möchten Sie senden?} withdraw {Wie viel möchten Sie abheben?} tip {Wie viel möchten Sie spenden?}}",
```
That should do it. Ü

PS: @Frooxius Is there a specific reason that the entire sentence uses the replacement system instead of just the changed word?
e.g. instead of above, `"How much do you want to {action,select, send {send} withdraw {withdraw} tip {tip}}?"`